src/iop/Permutohedral.h

```diff
@@ -135,13 +135,17 @@ template <int KD, int VD> class HashTablePermutohedral
     values = new Value[maxFill()] { 0 };
   }
 
+  HashTablePermutohedral(const HashTablePermutohedral &) = delete;
```

Comment:
This class manages dynamic memory. Therefore, the copy constructor and the copy assignment operator must be implemented or deleted. The compilers default implementation will not do.